package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Should the testing dependencies be in their own subfolder? We already have a lot of stuff in the top-level of the repo.